osu.Game/osu.Game.csproj

```diff
@@ -1,827 +1,41 @@
-      <Visible>False</Visible>
-      <ProductName>.NET Framework 3.5 SP1</ProductName>
-      <Install>false</Install>
-    </BootstrapperPackage>
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\osu-framework\osu.Framework\osu.Framework.csproj">
-      <Project>{c76bf5b3-985e-4d39-95fe-97c9c879b83a}</Project>
-      <Name>osu.Framework</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\osu-resources\osu.Game.Resources\osu.Game.Resources.csproj">
-      <Project>{d9a367c9-4c1a-489f-9b05-a0cea2b53b58}</Project>
-      <Name>osu.Game.Resources</Name>
-    </ProjectReference>
+    <Content Include="*.so">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="*.dylib">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="*.dll.config">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="Audio\SampleInfo.cs" />
-    <Compile Include="Audio\SampleInfoList.cs" />
-    <Compile Include="Beatmaps\Beatmap.cs" />
-    <Compile Include="Beatmaps\BeatmapConverter.cs" />
-    <Compile Include="Beatmaps\BeatmapDifficulty.cs" />
-    <Compile Include="Beatmaps\BeatmapInfo.cs" />
-    <Compile Include="Beatmaps\BeatmapManager.cs" />
-    <Compile Include="Beatmaps\BeatmapMetadata.cs" />
-    <Compile Include="Beatmaps\BeatmapMetrics.cs" />
-    <Compile Include="Beatmaps\BeatmapOnlineInfo.cs" />
-    <Compile Include="Beatmaps\BeatmapProcessor.cs" />
-    <Compile Include="Beatmaps\BeatmapSetFileInfo.cs" />
-    <Compile Include="Beatmaps\BeatmapSetInfo.cs" />
-    <Compile Include="Beatmaps\BeatmapSetOnlineInfo.cs" />
-    <Compile Include="Beatmaps\BeatmapStatistic.cs" />
-    <Compile Include="Beatmaps\BeatmapStore.cs" />
-    <Compile Include="Beatmaps\ControlPoints\ControlPoint.cs" />
-    <Compile Include="Beatmaps\ControlPoints\ControlPointInfo.cs" />
-    <Compile Include="Beatmaps\ControlPoints\DifficultyControlPoint.cs" />
-    <Compile Include="Beatmaps\ControlPoints\EffectControlPoint.cs" />
-    <Compile Include="Beatmaps\ControlPoints\SoundControlPoint.cs" />
-    <Compile Include="Beatmaps\ControlPoints\TimingControlPoint.cs" />
-    <Compile Include="Beatmaps\DifficultyCalculator.cs" />
-    <Compile Include="Beatmaps\Drawables\BeatmapBackgroundSprite.cs" />
-    <Compile Include="Beatmaps\Drawables\BeatmapGroup.cs" />
-    <Compile Include="Beatmaps\Drawables\BeatmapPanel.cs" />
-    <Compile Include="Beatmaps\Drawables\BeatmapSetCover.cs" />
-    <Compile Include="Beatmaps\Drawables\BeatmapSetHeader.cs" />
-    <Compile Include="Database\DatabaseContextFactory.cs" />
-    <Compile Include="Database\IHasPrimaryKey.cs" />
-    <Compile Include="Migrations\20171019041408_InitialCreate.cs" />
-    <Compile Include="Migrations\20171019041408_InitialCreate.Designer.cs">

```

Comment:
Could EF tools recognize these correctly?
And I'd want EF tools to generate them inside `Database` folder.